src/TypeDefinition.js

```diff
@@ -2,14 +2,16 @@
 
 import React from 'react';
 
-import {
-  Animated,
-} from 'react-native';
+// @todo figure out how to use Animated.Value type in web context
+type AnimatedValue = *;
 
-import type {
-  HeaderMode,
-  HeaderProps,
-} from './views/Header';
```

Comment:
That file is required by prop-types plugin everywhere and so, it cannot have any React Native related types in it (it will try to bundle react-native for web target otherwise).